Find a functionally equivalent code:com.android.internal.widget.LockPatternUtils.getKeyguardStoredPasswordQuality:COMMENT
Method Modifier: public      internal    
Comment:/**
 * Retrieves the quality mode we're in.
 * {@see DevicePolicyManager#getPasswordQuality(android.content.ComponentName)}
 *
 * @return stored password quality
 */

@@ -1,8 +1,3 @@
 {
-    int quality = (int) getLong(PASSWORD_TYPE_KEY, DevicePolicyManager.PASSWORD_QUALITY_SOMETHING);
-    // method and treat biometric as a special case.
-    if (quality == DevicePolicyManager.PASSWORD_QUALITY_BIOMETRIC_WEAK) {
-        quality = (int) getLong(PASSWORD_TYPE_ALTERNATE_KEY, DevicePolicyManager.PASSWORD_QUALITY_SOMETHING);
-    }
-    return quality;
+    return getKeyguardStoredPasswordQuality(getCurrentOrCallingUserId());
 }

